cmake/modules/RootConfiguration.cmake

```diff
@@ -275,6 +275,7 @@ set(gvizincdir ${GVIZ_INCLUDE_DIR})
 set(gvizcflags)
 
 set(buildpython ${value${pyroot}})
+set(pyrootexp ${pyroot_experimental})
```

Comment:
The changes go in the right direction, but you have to be careful. The the user may configure ROOT with `-Dpyroot_experimental=1` or `-Dpyroot_experimental=yes` or ... In this case the logic will fail because the replaced variable will not be `ON`. It is better like this:
```
if(pyroot_experimental)
  set(pyrootexp ON)
endif()
```

Reply:
Thank you @peremato, I added your fix. If the builds are ok I will merge today.